requests/packages/idna/codec.py

```diff
@@ -0,0 +1,118 @@
+from idna.core import encode, decode, alabel, ulabel, IDNAError
```

Comment:
Will this work? I think we need to rewrite this import to be an absolute relative import.